.circleci/config.yml

```diff
@@ -24,21 +24,29 @@ jobs:
               python -m venv venv || virtualenv venv
 
       - restore_cache:
-          key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "package.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
+          key: deps1-{{ .Branch }}-{{ checksum ".circleci/dev/dev-requirements.txt" }}-{{ checksum "package.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
 
       - run:
           name: Install dependencies
           command: |
               . venv/bin/activate
-              pip install -r requirements.txt
+              pip install -r .circleci/dev/dev-requirements.txt
               npm install --ignore-scripts
 
       - save_cache:
-          key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "package.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
+          key: deps1-{{ .Branch }}-{{ checksum ".circleci/dev/dev-requirements.txt" }}-{{ checksum "package.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
           paths:
               - "venv"
               - "node_modules"
 
+      - run:
+          name: prettier --list-different
```

Comment:
Why prettier ? It's auto generated code.

Reply:
True, this would probably just get annoying. I'll remove it